CONTRIBUTORS.txt

```diff
@@ -78,6 +85,7 @@ needs to be listed here.
 - Tim Condon <0xtimc@gmail.com>
```

Comment:
@broadwaylamb  if you would like me to add a proper email address here, please reply with the preferred email address.

Reply:
sure!
jaskiewiczs@icloud.com

Reply:
thanks, done.